swift-lang/pom.xml

```diff
@@ -12,7 +12,14 @@
     <dependencies>
 
         <dependency>
-            <groupId>io.ecocode</groupId>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>ecocode-rules-specifications</artifactId>
+            <version>1.5.0</version>
```

Comment:
no need to specify here de version ... use ${ecocode-rules-specifications.version} instead please

Reply:
right ! I fix it